feat(config): managerFilePatterns (#34615)
renovatebot/renovate: lib/modules/manager/helmsman/readme.md

The `helmsman` manager is currently limited and does not support the full feature set of [Helmsman](https://github.com/Praqma/helmsman), read about the limitations below.

### Non-configured filePatterns

By default the `helmsman` manager has an empty array for its `filePatterns` configuration option, because there is no convention for file naming in practice.
This means that `helmsman` won't search for any files, and you won't get any updates from the manager.

> To enable the `helmsman` manager, provide a valid `filePatterns` yourself, for example:

Reviewer: ```suggestion
To enable the `helmsman` manager, you must give Renovate a valid `filePatterns`.
For example:
```

Author: Former is good
